### ISSUE TITLE
How to add Jquery validation on symfony 4 repeated type Password field?

### ISSUE THREAD

**Issue by Sid993** [Sid993/Symfony#1]
`{% extends 'base.html.twig' %}

{% block title %}Register{% endblock %}

{% block stylesheets %}
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
	<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.7/jquery.js"></script> 
    <script src="http://malsup.github.com/jquery.form.js"></script> 
    <script src="http://ajax.aspnetcdn.com/ajax/jquery.validate/1.11.1/jquery.validate.min.js"></script>
	<link rel="stylesheet" href="/css/home.css"> 
	<link rel="stylesheet" href="/css/register.css"> 

	<script type="text/javascript">
		$().ready(function(){
			$("#register_form").validate({
				rules:{
       				email:{
       					required: true,
       					email: true
       				},
       				Password_first:{
       					required: true,
       					minlength: 6,
       					maxlength:10
       				},
       				Password_second:{
       					equalTo:"#Passwordfirst"
       				},
       			},
       			messages:{
       				email:"Please specify a valid email address",
       				Password_first:"Please enter at least 6 number",
       				Password_second:{
       					equalTo:"Password Does not Match"
       				}
       			},
       			highlight: function (element) {
		            $(element).closest('.form-group').addClass('has-error');
		        },
				submitHandler: function(form) {
					$.ajax({
					    type: "POST",
					    url: "ajax_submit",
					    data: $("#register_form").serialize(),
					    contentType: "application/json; charset=utf-8",
					    dataType: "json",
					    async: true,
					    success: function(response){
					    	if(response.Success){
					    		$("#register_form").trigger("reset");
					    		$("#success_msg").append("<p>"+response.Message+"</p>");
					    		setTimeout(function(){ window.location.replace("/profile"); }, 1000);
					    	}	
					    	else{
					    		$("#error_msg").append("<p>"+response.Message+"</p>");
					    	}
					    },
					    error: function(err){
					    	console.log(err);
					    }
					});
					return false;
				}
			});
		});
	</script>   
{% endblock %}

{% block body %}
<form class="my-form" role="form" id="register_form" method="POST">
	<div class="form-group">
       <h1 class="register">Register</h1>
       {{ form_errors(registrationForm) }}
        <div class="form-group">
        	{{ form_row(registrationForm.email) }}
        </div>
        <div class="form-group">
        	{{ form_row(registrationForm.Password) }}
        </div>
        <button name="submit" type="submit" class="button" id="register_button">Register</button>
       {{ form_end(registrationForm) }}
       <span id="success_msg" class="success"></span>
       <span id="error_msg" class="error"></span>
   </div>
</form>
    
{% endblock %}

//Formbuilder file
<?php

namespace App\Form;

use App\Entity\User;
use Symfony\Component\Form\AbstractType;
use Symfony\Component\Form\Extension\Core\Type\PasswordType;
use Symfony\Component\Form\Extension\Core\Type\CheckboxType;
use Symfony\Component\Form\FormBuilderInterface;
use Symfony\Component\OptionsResolver\OptionsResolver;
use Symfony\Component\Validator\Constraints\NotBlank;
use Symfony\Component\Validator\Constraints\Length;
use Symfony\Component\Validator\Constraints\IsTrue;
use Symfony\Component\Form\Extension\Core\Type\RepeatedType;
use Symfony\Component\Form\Extension\Core\Type\EmailType;
use Symfony\Component\Validator\Constraints\Email;

class RegistrationFormType extends AbstractType
{
    public function buildForm(FormBuilderInterface $builder, array $options)
    {
        $builder
            ->add('email', EmailType::class, [
                'constraints' => [
                    new Email([
                        'message' => 'Please enter a valid email address.',
                    ])
                ]
            ])

            ->add('Password', RepeatedType::class, [
                // instead of being set onto the object directly,
                // this is read and encoded in the controller
                'type' => PasswordType::class,
                'required' => true,
                'mapped' => true,
                'constraints' => [
                    new NotBlank([
                        'message' => 'Please enter a password',
                    ]),
                    new Length([
                        'min' => 6,
                        'minMessage' => 'Your password should be at least 6 characters',
                        // max length allowed by Symfony for security reasons
                        'max' => 10,
                    ]),
                ],
                'first_options' => ['label' => 'Password'],
                'second_options' => ['label' => 'Confirm Password'],
                'invalid_message' => [ 'Your password does not match the confirmation.' ]
            ]);
    }

    public function configureOptions(OptionsResolver $resolver)
    {
        $resolver->setDefaults([
            'data_class' => User::class,
        ]);
    }
}
`

**Comment by Sid993** [comment]
rule name same as input type field name
